### DISCUSSION TITLE
EZVIZ CP4 Spioncino Digitale

### DISCUSSION THREAD

**Discussion by gideonklein** [indomus/forum#2325] [HUB: Home Assistant]
C'è un modo per integrare EZVIZ CP4 Spioncino Digitale in HA? Ho capito che non ha il flusso rtsp, ma tramite intergrazione Ezviz cloud è possibile? Non ho trovato niente online. Grazie in anticipo

**Comment by KC-inDomus** [comment]
> C'è un modo per integrare EZVIZ CP4 Spioncino Digitale in HA? Ho capito che non ha il flusso rtsp, ma tramite intergrazione Ezviz cloud è possibile? Non ho trovato niente online. Grazie in anticipo

purtroppo no

**Reply by gideonklein** [reply]
sospettavo. Grazie!

**Comment by cldparisi** [comment]
Ho stesso problema con lo spioncino Ezviz DP2C, sapete si anche questo non ha supporto rtsp?
Non c'è altro modo per integrare la cam del DP2C

**Reply by galamarco** [reply]
come hai integrato il dispositivo DP2C in Home Assistant? Io l'ho fatto con l'integrazione ufficiale, funzionano tutti i sensori tranne il flusso video che non si vede mai.
Il risultato è che so tutto quello che accade fuori dalla porta ma non vedo nulla. 
Ho provato anche a integrare il dispositivo via yaml come segue ma nulla...
```
camera:
   - platform: ffmpeg
     name: Spioncino camera
     input: rtsp://admin:PASSWORD@192.168.6.169:554/H.264
```

**Reply by galamarco** [reply]
@M0ntyC0r4 temevo questa risposta... che poi è perfettamente in linea con quanto hanno già fatto su tutte le telecamere interne...

Dove si trova il firmware per fare il downgrade?

**Reply by gideonklein** [reply]
Per quanto riguarda la batteria che si scarica, io ho risolto il problema mettendo un piccolo ups all'interno del blindato sempre alimentato dai contatti della porta quando si chiude (come fosse il contatto apriporta). Così ho lo spioncino sempre alimentato anche con la porta aperta. L'ups interviene in tempo reale quando non alimentato. Non ho Ezviz ma altro spioncino senza schermo che invia il flusso rtsp e quindi integrato in HA. Prenderò Ezviz perchè ha lo schermo se con il downgrade del firmware permette di avere il flusso rtsp, poi lo renderò sveglio con qualche automazione costante. Cosa ne pensate?

**Reply by gideonklein** [reply]
![1711792663006](https://github.com/indomus/forum/assets/79428290/5a46d4a5-8e58-4dee-88e9-58eb0d207514)
![1711792663015](https://github.com/indomus/forum/assets/79428290/0fb8e177-6983-4fc4-aca5-c7df80fd1a90)

Capisci ora?

**Reply by gideonklein** [reply]
![1711820348655](https://github.com/indomus/forum/assets/79428290/53500ef6-a9f4-4917-ae22-e864640ba2f8)
Ovviamente ci deve essere un cavo che trasporta la 5v che deve arrivare nel telaio della porta, quindi trasformatore a monte. I profili solitamente hanno dei buchi coperti da coperchi che si possono togliere. Poi passare la sonda. Io sono stato fortunato perchè ho gli interruttori vicino il telaio (come da foto), quindi dalla scatola 503 ho fatto un foro che è arrivato al telaio. Spero di essere stato chiaro

**Reply by gideonklein** [reply]
Ups da 5v ne trovi quanti ne vuoi anche su aliexpress o simili

**Reply by gideonklein** [reply]
![Screenshot 2024-03-30 184612](https://github.com/indomus/forum/assets/79428290/d55e3789-6dc4-4716-80fb-9965d0bb42e1)
Spioncino a porta aperta. Saluti

**Reply by gideonklein** [reply]
Niente corrugato, devi fare un foro con una punta lunga all'interno dalla tua scatola di derivazione al telaio della porta (sempre all'interno). Canaline esterne o fili esterni non sono a norma e pericolosi. Se non puoi fare il buco dal dalla scatola è preferibile fare un solco nel muro che arrivi al telaio e alla scatola, mettere un corrugato e ricoprire il muro e pitturare la parte fatta. Tutti lavoretti con il fai da te. Ma se non sei capace, chiamo un'elettricista.
Ups vanno bene tutti, non troppo grandi di dimensione fisica. Io avevo questo e va bene:
![51HJ8NyqFgL _AC_SL1001_](https://github.com/indomus/forum/assets/79428290/5f178cd8-f8c0-49f3-8e99-710df744afdf)

Mi pare che ho detto tutto. Per altre info, chiama un elettricista
Saluti

**Reply by gideonklein** [reply]
è indifferente, io avevo la 503 vicino e tu hai la scatola di derivazione, entrambi hanno già il neutro dove devi collegare il trasformatore da 230v a 5v DC, non puoi portare la 230 al telaio in metallo della porta. Questi sono i primi rudimenti. Dalle tue risposte, capisco che non hai minimamente a che fare con l'elettricità, quindi non lo puoi fare da solo. DEVI CHIAMARE un'elettricista. Con la 230v non si scherza, non ci si può improvvisare e non ti rispondo piu. Non voglio essere complice di eventuali infortuni che possono portare alla morte. Fai da te si ma quando si è consapevoli dei propri limiti

**Reply by indomus** [reply]
Bestemmiare sul nostro forum, scordatelo. Già avvisato più volte, ora bannato.
